diff --git a/externals/shared/lib/zgui/build.zig.zon b/externals/shared/lib/zgui/build.zig.zon
index d94ff7cd..c7c9a8f2 100644
--- a/externals/shared/lib/zgui/build.zig.zon
+++ b/externals/shared/lib/zgui/build.zig.zon
@@ -18,8 +18,7 @@
             .lazy = true,
         },
         .zglfw = .{
-            .url = "https://github.com/zig-gamedev/zglfw/archive/0dd29d8073487c9fe1e45e6b729b3aac271d5a71.tar.gz",
-            .hash = "zglfw-0.10.0-dev-zgVDNIG4IQBWN_sfMD-xfC9bJS2hbBN2W7jNlDLovcdC",
+            .path = "../zglfw",
             .lazy = true,
         },
         .zgpu = .{
