diff --git a/externals/shared/lib/zgui/build.zig.zon b/externals/shared/lib/zgui/build.zig.zon
index d94ff7cd..c7c9a8f2 100644
--- a/externals/shared/lib/zgui/build.zig.zon
+++ b/externals/shared/lib/zgui/build.zig.zon
@@ -18,8 +18,7 @@
             .lazy = true,
         },
         .zglfw = .{
-            .url = "https://github.com/zig-gamedev/zglfw/archive/feeeff9386dd44ba2c18488113023865c41b80a4.tar.gz",
-            .hash = "zglfw-0.10.0-dev-zgVDNLyhIQCZR_rbIf7hnCKuifzcmPOLOQhd_hrqq8cl",
+            .path = "../zglfw",
             .lazy = true,
         },
         .zgpu = .{
